index.js

```diff
@@ -54,10 +54,11 @@ function getChromeExe(chromeDirName) {
 
   for (i = 0; i < prefixes.length; i++) {
     prefix = prefixes[i];
-    if (fs.realpathSync(prefix + suffix)) {
-      windowsChromeDirectory = prefix + suffix;
-      break;
-    }
+    try {
+      windowsChromeDirectory = path.join(prefix, suffix);
+      fs.accessSync(windowsChromeDirectory);
```

Comment:
`fs.F_OK` is the default mode for `accessSync`, which is what we need to check if the file (Windows) folder (OS X) exists https://nodejs.org/api/fs.html#fs_fs_access_path_mode_callback